scipy/special/log_iv.py

```diff
@@ -0,0 +1,43 @@
+"""
+Log of Bessel functions
+
+References
+----------
+.. [dlmf] NIST Digital Library of Mathematical Functions
+       https://dlmf.nist.gov/10.41#ii
+.. [a144617] Sequence A144617, The On-Line Encyclopedia of Integer Sequences,
+       https://oeis.org/A144617
+"""
+#
+# Author:  Seth Troisi 2020
+
+from numpy import log
+from scipy.special import iv, ive
+from ._logiv import log_iv_asym
+
+__all__ = ['logiv', 'logive']
+
+def logiv(v, z):
```

Comment:
Maybe move the dispatch to Cython, so that cython_special interface is consistent.